lib/install/management/delete_test.go

```diff
@@ -103,7 +104,7 @@ func createAppliance(ctx context.Context, sess *session.Session, conf *config.Vi
 		isVC:    sess.IsVC(),
 		force:   false,
 	}
-	delete(conf.Networks, "bridge") // FIXME: cannot create bridge network in simulator
+
 	if d.isVC {
 		if d.vchVapp, err = d.createVApp(conf, vConf); err != nil {
 			// FIXME: Got error: ServerFaultCode: ResourcePool:resourcepool-14 does not implement: CreateVApp. Simulator need to implement CreateVApp
```

Comment:
I left it for now since CreateVApp still isn't implement by the simulator.  Mentioned in the issue: https://github.com/vmware/vic/issues/1859#issuecomment-275544136